app/services/preserved_object_handler.rb

```diff
@@ -60,7 +60,19 @@ def create
   end
 
   def check_existence
-    # this is just a stub
+    if invalid?
+      handler_results.add_result(PreservedObjectHandlerResults::INVALID_ARGUMENTS, errors.full_messages)
+    elsif PreservedObject.exists?(druid: druid)
+      confirm_version
+      if handler_results.contains_result_code? PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT
+        update_version_after_validation
```

Comment:
Should we just make confirm_version return a value for you to use so we don't have to search our own results? I realize we talked about this, but now that I see the code, it just seems a little weird.

Reply:
i sorta like that we can just examine the results we've already collected, so i think i have a slight preference for this, but i'd be totally fine with switching if you have a strong preference for using a return value, i'd be open to switching.

actually, one other thing in favor of this, is that `confirm_version` already returns the result array, which is structurally consistent with other similar methods, which i like.  we could look at that result array, but that's actually what tony and i had started out doing, and it looked a little messy, and so we ended up at this new `contains_result_code?` method.

Reply:
`confirm_version` and all the similar public methods must return the results array so we can turn them into ReST calls with meaningful HTTP response codes for errors.  That's what started us down this whole results array approach in the first place.